### PR TITLE
i18n(es): update to `configuration-reference`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content

#### Description

- What does this PR change? Small fixes to `configuration-reference.md`

Please label this as `hacktoberfest-accepted`

### CODE REVIEW

File: src/pages/es/reference/configuration-reference.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ La URL final donde se desplegará. Astro usa esta URL completa para generar el s
 **Tipo:** `string`
 </p>
 
-La ruta base en la que se desplegará. Astro coincidirá esta ruta durante el desarrollo para que tu experiencia de desarrollo coincida con el entorno de desarrollo lo más cercano posible. En el siguiente ejemplo, `astro dev` iniciará el servidor en `/docs`.
+La ruta base en la que se desplegará. Astro compilará tus páginas y recursos usando esta ruta como la raiz. En este momento, esto no tiene efecto durante el desarrollo.
```
Comment (kevinzunigacuellar) [inline src/pages/es/reference/configuration-reference.md:126]:
```suggestion
La ruta base en la que se desplegará. Astro compilará tus páginas y recursos usando esta ruta como la raiz. En este momento, esta configuración no tiene efecto durante el modo de desarrollo.
```

File: src/pages/es/reference/configuration-reference.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ La URL final donde se desplegará. Astro usa esta URL completa para generar el s
 **Tipo:** `string`
 </p>
 
-La ruta base en la que se desplegará. Astro coincidirá esta ruta durante el desarrollo para que tu experiencia de desarrollo coincida con el entorno de desarrollo lo más cercano posible. En el siguiente ejemplo, `astro dev` iniciará el servidor en `/docs`.
+La ruta base en la que se desplegará. Astro compilará tus páginas y recursos usando esta ruta como la raiz. En este momento, esta configuración no tiene efecto durante el modo de desarrollo.
```
Comment (AbrahamX3) [inline src/pages/es/reference/configuration-reference.md:126]:
```suggestion
La ruta base en la que se desplegará. Astro compilará tus páginas y recursos usando esta ruta como la raíz. En este momento, esta configuración no tiene efecto durante el modo de desarrollo.
```

**Review by AbrahamX3 — CHANGES_REQUESTED** [review]
Que tal, deje una corrección 🙌

**Comment by kevinzunigacuellar** [comment]
gracias @dreyfus92 y @AbrahamX3 🚀

**Pull request merged**